[2024-06-21 23:56:35,880: INFO: main: WELCOME]
[2024-07-03 15:47:44,015: INFO: main: {'Data Ingestion Stage'}]
[2024-07-03 15:47:44,016: ERROR: main: DataIngestionTrainingPipeline.main() missing 1 required positional argument: 'self']
Traceback (most recent call last):
  File "c:\Users\heman\OneDrive\Desktop\Text Summariser\main.py", line 8, in <module>
    data_ingestion.main()
TypeError: DataIngestionTrainingPipeline.main() missing 1 required positional argument: 'self'
[2024-07-03 15:56:30,330: INFO: main: {'Data Ingestion Stage'}]
[2024-07-03 15:56:30,330: ERROR: main: DataIngestionTrainingPipeline.main() missing 1 required positional argument: 'self']
Traceback (most recent call last):
  File "c:\Users\heman\OneDrive\Desktop\Text Summariser\main.py", line 8, in <module>
    data_ingestion.main()
TypeError: DataIngestionTrainingPipeline.main() missing 1 required positional argument: 'self'
[2024-07-03 15:59:26,888: INFO: main: {'Data Ingestion Stage'}]
[2024-07-03 15:59:26,888: ERROR: main: DataIngestionTrainingPipeline.main() missing 1 required positional argument: 'self']
Traceback (most recent call last):
  File "c:\Users\heman\OneDrive\Desktop\Text Summariser\main.py", line 8, in <module>
    data_ingestion.main()
TypeError: DataIngestionTrainingPipeline.main() missing 1 required positional argument: 'self'
[2024-07-03 15:59:35,317: INFO: main: {'Data Ingestion Stage'}]
[2024-07-03 15:59:35,318: ERROR: main: name 'self' is not defined]
Traceback (most recent call last):
  File "c:\Users\heman\OneDrive\Desktop\Text Summariser\main.py", line 8, in <module>
    data_ingestion.main(self)
                        ^^^^
NameError: name 'self' is not defined
[2024-07-03 16:01:30,136: INFO: main: {'Data Ingestion Stage'}]
[2024-07-03 16:01:30,136: ERROR: main: DataIngestionTrainingPipeline.main() missing 1 required positional argument: 'self']
Traceback (most recent call last):
  File "c:\Users\heman\OneDrive\Desktop\Text Summariser\main.py", line 8, in <module>
    data_ingestion.main()
TypeError: DataIngestionTrainingPipeline.main() missing 1 required positional argument: 'self'
[2024-07-03 16:01:58,730: INFO: main: {'Data Ingestion Stage'}]
[2024-07-03 16:01:58,731: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-03 16:01:58,733: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 16:01:58,733: INFO: common: created directory at: artifacts]
[2024-07-03 16:01:58,734: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-03 16:02:01,597: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 3163:36F5C4:8F8CC:B9FFE:6685289E
Accept-Ranges: bytes
Date: Wed, 03 Jul 2024 10:31:59 GMT
Via: 1.1 varnish
X-Served-By: cache-hyd1100025-HYD
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1720002719.084788,VS0,VE720
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: f8a4b562bb808396206226b846df3ef1659b1992
Expires: Wed, 03 Jul 2024 10:36:59 GMT
Source-Age: 0

]
[2024-07-03 16:02:01,744: INFO: main: COMPLETED]
[2024-07-03 16:02:23,278: INFO: main: {'Data Ingestion Stage'}]
[2024-07-03 16:02:23,280: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-03 16:02:23,281: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 16:02:23,282: INFO: common: created directory at: artifacts]
[2024-07-03 16:02:23,282: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-03 16:02:26,166: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 3163:36F5C4:8F8CC:B9FFE:6685289E
Accept-Ranges: bytes
Date: Wed, 03 Jul 2024 10:32:23 GMT
Via: 1.1 varnish
X-Served-By: cache-hyd1100020-HYD
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1720002743.358593,VS0,VE1
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: e7c8e94b9d6aa55d49beabb11e04e091fe9c7772
Expires: Wed, 03 Jul 2024 10:37:23 GMT
Source-Age: 24

]
[2024-07-03 16:02:26,312: INFO: main: COMPLETED]
[2024-07-03 16:22:53,660: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-03 16:22:53,672: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 16:22:53,673: INFO: common: created directory at: artifacts]
[2024-07-03 16:29:49,283: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-03 16:29:49,293: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 16:29:49,294: INFO: common: created directory at: artifacts]
[2024-07-03 16:29:49,296: INFO: common: created directory at: artifacts/data_validation]
[2024-07-03 23:12:14,021: INFO: main: {'Data Validation Stage'}]
[2024-07-03 23:12:14,021: ERROR: main: name 'DataValidationTrainingPipeline' is not defined]
Traceback (most recent call last):
  File "c:\Users\heman\OneDrive\Desktop\Text Summariser\main.py", line 8, in <module>
    data_validation = DataValidationTrainingPipeline()
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'DataValidationTrainingPipeline' is not defined. Did you mean: 'DataIngestionTrainingPipeline'?
[2024-07-03 23:12:51,867: INFO: main: {'Data Validation Stage'}]
[2024-07-03 23:12:51,867: ERROR: main: type object 'DataValiadtion' has no attribute 'main']
Traceback (most recent call last):
  File "c:\Users\heman\OneDrive\Desktop\Text Summariser\main.py", line 9, in <module>
    data_validation .main()
    ^^^^^^^^^^^^^^^^^^^^^
AttributeError: type object 'DataValiadtion' has no attribute 'main'
[2024-07-03 23:12:59,455: INFO: main: {'Data Validation Stage'}]
[2024-07-03 23:12:59,456: ERROR: main: DataValiadtion.__init__() missing 1 required positional argument: 'config']
Traceback (most recent call last):
  File "c:\Users\heman\OneDrive\Desktop\Text Summariser\main.py", line 8, in <module>
    data_validation = DataValiadtion()
                      ^^^^^^^^^^^^^^^^
TypeError: DataValiadtion.__init__() missing 1 required positional argument: 'config'
[2024-07-03 23:14:23,334: INFO: main: {'Data Validation Stage'}]
[2024-07-03 23:14:23,335: ERROR: main: name 'DataValiadtionTrainingPipeline' is not defined]
Traceback (most recent call last):
  File "c:\Users\heman\OneDrive\Desktop\Text Summariser\main.py", line 8, in <module>
    data_validation = DataValiadtionTrainingPipeline()
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'DataValiadtionTrainingPipeline' is not defined. Did you mean: 'DataIngestionTrainingPipeline'?
[2024-07-03 23:15:22,528: INFO: main: {'Data Validation Stage'}]
[2024-07-03 23:15:22,529: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-03 23:15:22,531: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 23:15:22,531: INFO: common: created directory at: artifacts]
[2024-07-03 23:15:22,532: INFO: common: created directory at: artifacts/data_validation]
[2024-07-03 23:15:22,532: ERROR: main: name 'DataValidation' is not defined]
Traceback (most recent call last):
  File "c:\Users\heman\OneDrive\Desktop\Text Summariser\main.py", line 9, in <module>
    data_validation .main()
  File "C:\Users\heman\OneDrive\Desktop\Text Summariser\src\textsummariser\pipeline\stage2data_validation.py", line 13, in main
    data_validation = DataValidation(config=data_validation_config)
                      ^^^^^^^^^^^^^^
NameError: name 'DataValidation' is not defined. Did you mean: 'DataValiadtion'?
[2024-07-03 23:17:40,079: INFO: main: {'Data Validation Stage'}]
[2024-07-03 23:17:40,081: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-03 23:17:40,082: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 23:17:40,083: INFO: common: created directory at: artifacts]
[2024-07-03 23:17:40,084: INFO: common: created directory at: artifacts/data_validation]
[2024-07-03 23:17:40,084: ERROR: main: 'DataValidation' object has no attribute 'download_file']
Traceback (most recent call last):
  File "c:\Users\heman\OneDrive\Desktop\Text Summariser\main.py", line 9, in <module>
    data_validation .main()
  File "C:\Users\heman\OneDrive\Desktop\Text Summariser\src\textsummariser\pipeline\stage2data_validation.py", line 14, in main
    data_validation.download_file()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'DataValidation' object has no attribute 'download_file'
[2024-07-03 23:23:06,388: INFO: main: {'Data Validation Stage'}]
[2024-07-03 23:23:06,390: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-03 23:23:06,391: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 23:23:06,391: INFO: common: created directory at: artifacts]
[2024-07-03 23:23:06,392: INFO: common: created directory at: artifacts/data_validation]
[2024-07-03 23:23:06,393: ERROR: main: name 'DataValiadtion' is not defined]
Traceback (most recent call last):
  File "c:\Users\heman\OneDrive\Desktop\Text Summariser\main.py", line 9, in <module>
    data_validation .main()
  File "C:\Users\heman\OneDrive\Desktop\Text Summariser\src\textsummariser\pipeline\stage2data_validation.py", line 13, in main
    data_validation = DataValiadtion(config=data_validation_config)
                      ^^^^^^^^^^^^^^
NameError: name 'DataValiadtion' is not defined. Did you mean: 'DataValidation'?
[2024-07-03 23:23:33,947: INFO: main: {'Data Validation Stage'}]
[2024-07-03 23:23:33,949: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-03 23:23:33,950: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 23:23:33,950: INFO: common: created directory at: artifacts]
[2024-07-03 23:23:33,950: INFO: common: created directory at: artifacts/data_validation]
[2024-07-03 23:23:33,952: INFO: main: COMPLETED]
